node/toolrunner.ts

```diff
@@ -764,12 +771,12 @@ export class ToolRunner extends events.EventEmitter {
 
     /**
      * Pipe output of exec() to another tool
-     * @param tool
+     * @param tools
      * @param file  optional filename to additionally stream the output to.
      * @returns {ToolRunner}
      */
-    public pipeExecOutputToTool(tool: ToolRunner, file?: string): ToolRunner {
-        this.pipeOutputToTool = tool;
+    public pipeExecOutputToTool(tools: ToolRunner[], file?: string): ToolRunner {
```

Comment:
I think we shouldn't break back compat here. We should still allow you to call `pipeExecOutputToTool(tool)` - that can turn around and immediately call `pipeExecOutputToTool([tool])